tests/unit/design-system/components/Chip.spec.jsx

```diff
@@ -0,0 +1,137 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import Chip from '~/design-system/components/chip/Chip'
+
+// Test for Basic Chip
```

Comment:
Text in `describe` method is understandable, so you don't need to write those comments in this file.